src/styles/utls/reset.scss

```diff
@@ -0,0 +1,5 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
add empty line it he end of file

Reply:
I have empty line in my code, but it wasn't pushed to the github